packages/dso-toolkit/components/Voorbeelden/10-Patronen/subtitel.config.yml

```diff
@@ -0,0 +1,25 @@
+notes: |
+status: wip
+context:
+  menu:
```

Comment:
Waarom geef je deze context mee aan `@header` ? Is het van vitaal belang voor het voorbeeld?

Reply:
Blijkbaar niet, de doorbloeding van het origineel is exact hetzelfde